### ISSUE TITLE
Co-simulation crashes when using strings in Overture FMU

### ISSUE THREAD

**Issue by kgpierce** [into-cps/intocps-ui#165]
I have a multi-model that passes strings between VDM FMUs that crashes the JVM after around 0.5 simulated seconds:

```
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x000000007138b8f0, pid=3904, tid=0x00000000000018bc
#
# JRE version: Java(TM) SE Runtime Environment (8.0_101-b13) (build 1.8.0_101-b13)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.101-b13 mixed mode windows-amd64 compressed oops)
# Problematic frame:
# V  [jvm.dll+0x27b8f0]
#
# Failed to write core dump. Minidumps are not enabled by default on client versions of Windows
#
# An error report file with more information is saved as:
# C:\Users\ken\Downloads\hs_err_pid3904.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
#

java.lang.NullPointerException at org.intocps.orchestration.coe.scala.ResultLogger$$anonfun$9$$anonfun$apply$6.apply(ResultLogger.scala:151)
```

This occurs on Windows and Mac, latest release / COE 0.15. 

The model can be found here: https://github.com/into-cps/case-study_uav_swarm/tree/kgp/ether and run using multi-model "5-UAV-Ether-Non-3D".

**Comment by lausdahl** [comment]
please retry with release 0.0.9 from the app and report back

also include the log file mentioned in the crash. Especially around the native stack frames.

**Comment by ldcouto** [comment]
Any news on this @kgpierce ?

**Comment by lausdahl** [comment]
This is work in progress. The issue is not yet fixed.

**Comment by ldcouto** [comment]
Thanks Kenneth. Updated label and milestone

**Comment by ldcouto** [comment]
Any news on this?

**Comment by lausdahl** [comment]
Fixed in COE 0.1.10
